packages/@aws-cdk/aws-stepfunctions/lib/states/state.ts

```diff
@@ -24,6 +24,13 @@ export interface StateProps {
      */
     inputPath?: string;
 
+    /**
+     * Parameters pass a collection of key-value pairs, either static values or JSONPath expressions that select from the input.
+     *
+     * @default No parameters
+     */
+    parameters?: object;
```

Comment:
Maybe this should be modeled as `{ [name: string]: string }`? Or at least `{ [name: string]: any }` if the values can also be non-string literals - that'd be somewhat more informative than `object`.